.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,62 @@
+name: aws-deploy
+
+on:
+  push:
+    branches: [ '*' ]
+    tags: [ 'v*' ]
```

Comment:
let's make this regex a tighter match, try something like this `v[0-9]+\.[0-9]+\.[0-9]+$`